packages/@aws-cdk/aws-signer/README.md

```diff
@@ -9,6 +9,14 @@
 >
 > [CFN Resources]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib
 
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development.
+> They are subject to non-backward compatible changes or removal in any future version. These are
+> not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be
+> announced in the release notes. This means that while you may use them, you may need to update
+> your source code when upgrading to a newer version of this package.
+
 ---
 
 <!--END STABILITY BANNER-->
```

Comment:
Could you add a section about AWS signer here? Here's a recent example from [Cognito](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-cognito/README.md#user-pools).

You should add information here about signing profile and profile versions.